src/dev-app/dev-app/routes.ts

```diff
@@ -85,6 +85,7 @@ export const DEV_APP_ROUTES: Routes = [
   {path: 'mdc-table', loadChildren: 'mdc-table/mdc-table-demo-module#MdcTableDemoModule'},
   {path: 'mdc-tabs', loadChildren: 'mdc-tabs/mdc-tabs-demo-module#MdcTabsDemoModule'},
   {path: 'menu', loadChildren: 'menu/menu-demo-module#MenuDemoModule'},
+  {path: 'overlay', loadChildren: 'overlay/overlay-demo-module#OverlayDemoModule'},
```

Comment:
We already have a route that shows the connected overlay (see `connected-overlay` a bit further down).

Reply:
Should I make this example a bit more advanced? For example a custom directive using the Overlay classes to implement a context menu in a list or a timed pop-up?

Reply:
I think the example is fine, we should just combine it with the previous demo under a single route so everything is in the same place. The current demo is used for debugging the positioning logic and it's not exposed on the docs site.

Reply:
Gotcha. Sounds good